### ISSUE TITLE
Check labels for creating and editing posts

### ISSUE THREAD

**Issue by Angamanga** [ushahidi/platform#4081]
There are two ways labels are handled in the platform, depending on the markup of the form:

```
<label for="username">Username</label>
<input id="username" type="text" />
```
and

```
<label>Username
     <input id="username" type="text" />
</label>
```

The form that is used for creating and editing posts are divided into a number of angular-directives. Go through the templates (html-files) in the following folder https://github.com/ushahidi/platform-client/tree/develop/app/main/posts/modify  and make sure the element is either using the for-attribute to associate the form-control or is encapsulating the form-control like the second example.

The  views are found by clicking the + icon in map or data-view:
Adding a new post:

<img width="199" alt="Screenshot 2020-10-05 at 12 27 46" src="https://user-images.githubusercontent.com/8624777/95068925-45f7d480-0706-11eb-8450-2d8e85010d53.png">

<img width="950" alt="Screenshot 2020-10-05 at 12 28 16" src="https://user-images.githubusercontent.com/8624777/95068917-42fce400-0706-11eb-8f00-b9e668e4d4f9.png">

Editing an existing post (you need to be logged in):
Go to dataview:
<img width="90" alt="Screenshot 2020-10-05 at 12 29 07" src="https://user-images.githubusercontent.com/8624777/95069010-64f66680-0706-11eb-9df1-02fa5d99efef.png">

Select a post and click edit:

<img width="1375" alt="Screenshot 2020-10-05 at 12 29 55" src="https://user-images.githubusercontent.com/8624777/95069157-953e0500-0706-11eb-99ff-0bf2f78d081e.png">

<img width="1296" alt="Screenshot 2020-10-05 at 12 30 50" src="https://user-images.githubusercontent.com/8624777/95069183-9ec76d00-0706-11eb-9a9a-9ce44ec8fb28.png">

 
If no adjustments is needed, just comment in this issue that you have worked through the issues and checked, if adjustment is needed, make the necessary changes and create a PR 🎉   If changes are needed, make sure the form still looks the same as before and that is has no effect on the visual appearance of the form.

Aha! Link: https://ushahiditeam.aha.io/features/PROD-837

**Comment by Red-0111** [comment]
@Angamanga Please assign will try my best

**Comment by Angamanga** [comment]
@Red-0111 Thank you for your interest! Please go ahead!

**Comment by Red-0111** [comment]
can u please add hacktoberfest label

**Comment by TunrayoIlawole** [comment]
Hi @Angamanga Can i work on this?

**Comment by Angamanga** [comment]
@TunrayoIlawole I assigned you to this one: https://github.com/ushahidi/platform/issues/3943

**Comment by Ifycode** [comment]
@Angamanga please can I contribute to this?

**Comment by rowasc** [comment]
hey @Ifycode this one is already taken , but I'm adding more issues so we should have more issues available soon today if you havent been able to find any

**Comment by bimbolabuari** [comment]
@Angamanga  @rowasc please can I contribute to this?

**Comment by rowasc** [comment]
hey @Red-0111 how are you doing with this ticket?

**Comment by Red-0111** [comment]
Sry was not able to do

**Comment by rowasc** [comment]
all good! Un-assigning.

**Comment by rowasc** [comment]
@Angamanga @renujain31  there may be overlap between this and the new accessibility tickets created? Unsure if I should close or merge them and I bet you both know better than me.

**Comment by renujain31** [comment]
Hey, As far as I know, the issues are completely different and there is no overlap between accessibility tickets that I created and this one but the files involved might be the same. Though, I don't think this might be any problem.

**Comment by rowasc** [comment]
ah alright. Wanted to make sure :) 
Thanks

**Comment by Angamanga** [comment]
@bimbolabuari this one is about the same as you already started. Please have a look and add to your PR if there is something more to check/add 🙏

**Comment by bimbolabuari** [comment]
Okay,I will check it up @Angamanga
Thank you!
